### ISSUE TITLE
LRS3 training

### ISSUE THREAD

**Issue by sungnyun** [Sreyan88/LipGER#4]
Hello, thank you for sharing the great work.

I have a question on the configuration of LRS3 training dataset.

In the paper, you have mentioned that 407 hours of LRS3 is used for training, and I found out that there are two files:
- lrs3_full_trainval_*.json
- lrs3_full_train_*.json

in the LipHyp dataset you've shared.

Which one is the trainset you used in the paper? I suppose lrs3_full_trainval_*.json is the trainset, right? Then, what is the validation set?

Thank you.

**Comment by Sreyan88** [comment]
Thank You for the question! This is an old project, and I do not remember much about it. But I will dig into the datasets to give you a clear answer asap. Thank You for your patience!
